docs/datamodel/index.rst

```diff
@@ -65,6 +65,11 @@ aspects.  Additionally, for instances of the child type
 instances of the child type are included in the set of all instances of
 the parent type.
 
+Multiple inheritance allows composing several types into one. A common
+pattern is to have many basic abstract types (such as ``Named``,
+``HasEmail``, ``HasAddress``, etc.) each with their own links and
+properties and then extending different combinations of them.
```

Comment:
Add a link to polymorphic queries, please

Reply:
Also worth mentioning that it's possible to inheriting from non-abstract types too.